versions/fix.ver

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # Fix file subfolder versions
 
-export aer_ver=20220805
+export aer_ver=
```

Comment:
```suggestion
export aer_ver=20250529
```

Reply:
HI, Kate:
Both versions are needed at this moment. 20220805 is for the 2014-2023 mean MERRA2 aerosol  and 20250529 is for the 6 year sliding MERRA2 aerosol as indicated by lines 530-556 in ush/forecast_predet.sh

Reply:
or put the files in 20220805 into 20250529.

Reply:
@AnningCheng-NOAA A couple questions:

1. Is this a temporary need?
2. Could the two versions be combined in a way to support both needs?

The `aer_ver` fix value is used to build the `fix/aer` folder (symlink to `$FIX_DIR/aer/$aer_ver` so we can only have one value, particularly once the system is in ops.

Reply:
@KateFriedman-NOAA it is probably not a temporary need.  Let us combine it by moving all the files in 20220805 to 20250529. I have change the scripts accordingly, but do not have the permission to move the files.